utils/build-click.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+KAIDAN_SOURCES=$(dirname "$(readlink -f "${0}")")/..
+
+ARCH=$(dpkg-architecture -qDEB_HOST_ARCH)
+DEB_HOST_MULTIARCH=$(dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+CLICK_TARGET_DIR="$KAIDAN_SOURCES/bin/ubuntu-touch/tmp/"
+
+! [ -d $CLICK_TARGET_DIR ] && mkdir $CLICK_TARGET_DIR
```

Comment:
Better just use `mkdir -p ${CLICK_TARGET_DIR}`

Reply:
But that will still fail if the directory already exists.

Reply:
![image](https://user-images.githubusercontent.com/13557034/36950330-c45da080-1ff4-11e8-8daa-dcf7fc65bb8b.png)
I don't think so.

Reply:
It doesn't fail